packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
+        const {locator} = this.state;
```

Comment:
nit: convention in the rest of the Blueprint codebase is to add spaces within the brackets when deconstructing: `{ locator }`. @adidahiya is there a `tslint` rule or existing issue for this, or should I file a FR?